db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js

```diff
@@ -344,6 +344,133 @@ SELECT
 FROM nokia_cm."LNCEL" t1
 `
 
+const COMBINED_KEY_PARAMAETERS = `
+--KEY PARAMETERS (3G Huawei CFGMML)
+select
+t1.data->>'DATETIME' as "VARDATE",
+'null' as "STATE",
+'null' as "REGIONAL",
+'HUAWEI' as"VENDOR",
+'3G' as "TECH",
+t2.data->>'SYSOBJECTID' as "NENAME",
+t1.data->>'BSCID' as "NEID",
+'null' as "SITEPROP",
+t3.data->>'NODEBID' as "SITE  ID",
+t1.data->>'NODEBNAME' as "SITENAME",
+t1.data->>'CELLID' as "CELLID",
+t1.data->>'CELLNAME' as "CELLNAME",
+t1.data->>'LOCELL' as "CI",
+'null' as "ACTSTATUS",
+'null' as "BLKSTATUS",
+'null' as "DLBANDWIDTH",
+t1.data->>'BANDIND' as "BAND",
+'null' as "CARR",
+t1.data->>'UARFCNDOWNLINK' as "DLF",
+t1.data->>'UARFCNUPLINK' as "ULF",
+t5.data->>'MCC' as "MCC",
+t5.data->>'MNC' as "MNC",
+hex_to_int(REPLACE(t1.data->>'LAC','H''','')) as "LAC",
+hex_to_int(REPLACE(t1.data->>'RAC','H''','')) as "RAC",
+CONCAT(t5.data->>'MCC', '-', t5.data->>'MNC', '-', hex_to_int(REPLACE(t1.data->>'LAC','H''','')), '-', t1.data->>'LOCELL') AS "CGI" ,
+'null' as "2G_BCCHNO",
+'null' as "2G_BSIC",
+'null' as "2G_TRX",
+t1.data->>'PSCRAMBCODE' AS "3G_PSC",
+'null' as "3G_DLCE",
+'null' as "3G_ULCE",
+'null' as "4G_PCI",
+'null' as "4G_TAC",
+'null' as "4G_ROOTIDX"
+FROM 
+huawei_cm."UCELL" t1 --where t1.data->>'FILENAME' LIKE 'CFGMML%'
+INNER JOIN huawei_cm."SYS" t2 on t1.data->>'FILENAME'=t2.data->>'FILENAME' and t1.data->>'BSCID'=t2.data->>'BSCID'
+INNER JOIN huawei_cm."UNODEB" t3 on t1.data->>'FILENAME'=t3.data->>'FILENAME' and t1.data->>'NODEBNAME'=t3.data->>'NODEBNAME'
+INNER JOIN huawei_cm."UCNOPERATOR" t5 ON t5.data->>'FILENAME' = t1.data->>'FILENAME'
+UNION
+--KEY PARAMETERS 3G Huawei NBI
+select
+t1.data->>'varDateTime' as "VARDATE",
+'null' as "STATE",
+'null' as "REGIONAL",
+'HUAWEI' as"VENDOR",
+'3G' as "TECH",
+t2.data->>'SYSOBJECTID' as "NENAME",
+t1.data->>'neid' as "NEID",
+'null' as "SITEPROP",
+t3.data->>'NODEBID' as "SITE  ID",
+t1.data->>'NODEBNAME' as "SITENAME",
+t1.data->>'CELLID' as "CELLID",
+t1.data->>'CELLNAME' as "CELLNAME",
+t1.data->>'LOCELL' as "CI",
+t1.data->>'ACTSTATUS' as "ACTSTATUS",
+t1.data->>'BLKSTATUS' as "BLKSTATUS",
+'null' as "DLBANDWIDTH",
+t1.data->>'BANDIND' as "BAND",
+'null' as "CARR",
+t1.data->>'UARFCNDOWNLINK' as "DLF",
+t1.data->>'UARFCNUPLINK' as "ULF",
+t5.data->>'MCC' as "MCC",
+t5.data->>'MNC' as "MNC",
+(t1.data->>'LAC')::INTEGER/1 as "LAC",
```

Comment:
Why divide LAC and RAC value by 1? Does it matter?

Reply:
Since LAC is in Hexadecimal value in CFGMML, need to mark it as integer